testsuite/WIKI.md

```diff
@@ -0,0 +1,198 @@
+The unit tests and the test data are bundled together in the package **MDAnalyisTests-_release_**. In order to run the tests, this package must be installed in addition to MDAnalysis.
+
+The tests also rely on the `pytest` and `numpy` packages, and require both to run.
+
+# Users
+Install [MDAnalysisTests](MDAnalysisTests) via
+```
+pip install --upgrade MDAnalysisTests
+```
+or download the tar file, unpack, and run `python setup.py install`.
+
+Run the tests by invoking
+```
+python -c 'from MDAnalysis.tests import test; test()'
+```
+
+All tests should pass (i.e. no **FAIL**, **ERROR**); *SKIPPED* or *XFAIL* are ok. For anything that fails or gives an error [ask on the user mailing list](http://users.mdanalysis.org) or [raise an issue](/MDAnalysis/mdanalysis/issues).
+
+# Developers #
+
+All tests should pass (i.e. no **FAIL**, **ERROR**); *SKIPPED* or *XFAIL* are ok. For anything that fails or gives an error **fix your code** (or *raise an issue*).
+
+**Do not push code that fails to the development branch.** Instead, push it to a feature or issue branch. It will run automatically through the unit tests by travis-ci and it will be available for comment and discussion by other developers.
+
+## Recommended ##
+Use the tests from the [git source repository](Source), which are located in the [testsuite/MDAnalysisTests](https://github.com/MDAnalysis/mdanalysis/tree/develop/testsuite) directory:
+```
+cd testsuite/MDAnalysisTests
+pytest  --disable-pytest-warnings --numprocesses 4
+```
+(Try increasing the number of processes; with 24 processes on 12 cores (+hyperthreading) this took ~40 seconds; in serial it takes ~30 min).
+
+To run specific tests just specify the path to the test file:
+```
+pytest test_analysis.py
+```
+
+Specific test classes inside test files, and even specific test methods, can also be specified:
+```
+# Test the entire TestContactMatrix class
+pytest test_analysis.py::TestContactMatrix
+# Test only test_sparse in the TestContactMatrix class
+pytest test_analysis.py::TestContactMatrix::test_sparse
```

Comment:
`test_analysis.py` no longer exists and should be replaced by some other test file in the examples (I'd use a test file in one of the subdirs, to highlight the fact that files must be specified by complete path -- when used like this pytest won't go find them under the test package for you).

As for the command-line syntax, I find it quite useful. Since I usually test away from the testsuite tree I again must specify complete paths:
```
pytest path/to/MDAnalysisTests/analysis/test_align.py::TestAlign::test_rmsd
```
Maybe an example like this could also be shown.